apps.yml

```diff
@@ -3369,15 +3369,17 @@ apps:
     - hris_is_staff
     - team_moco
     - team_mofo
-    authorized_users: []
-    client_id: 1B1ser8z1eyr4nOFtW1acz01i8GYIaBo
+    authorized_users:
```

Comment:
I suppose nowadays I would consider requesting an override group that we can hand over ownership/maintenance of back to the requesting party, but this is still fine.